addon/components/fm-field.js

```diff
@@ -19,10 +19,12 @@ export default Ember.Component.extend({
     if(!this.get('optionLabelPath')) {
       this.set('optionLabelPath', 'content.label');
     }
-    var dataAttributes = Object.keys(this.get('attrs')).filter(function(attr) {
-      return /data-/.test(attr);
-    });
-    this.set('dataAttributes', dataAttributes);
+    if(!Ember.isEmpty(this.get('attrs'))) {
+      var dataAttributes = Object.keys(this.get('attrs')).filter(function(attr) {
+        return /data-/.test(attr);
+      });
+      this.set('dataAttributes', dataAttributes);
+    }
```

Comment:
Necessary cause unit tests fails since this.get('attrs') was `null` and `Object.keys(null)` throws.